src/test/java/io/strimzi/kafka/bridge/http/ProducerIT.java

```diff
@@ -198,7 +203,7 @@ void sendSimpleMessageWithKey(VertxTestContext context) throws InterruptedExcept
     @Disabled("Will be check in the next PR, this is just external tests for Bridge")
     @DisabledIfEnvironmentVariable(named = "EXTERNAL_BRIDGE", matches = "((?i)TRUE(?-i))")
     @Test
-    void sendBinaryMessageWithKey(VertxTestContext context) throws InterruptedException, ExecutionException {
+    void sendBinaryMessageWithKey(VertxTestContext context) {
```

Comment:
you removed the throws but the test is disabled. Any reasons? I mean, is this change related to the PR? Are you going to check and enable this test as well?

Reply:
They were not used...so I have just removed them. I can rollback this change... )) because it's not related to this PR